packages/cli/src/commands/split/index.ts

```diff
@@ -264,15 +262,17 @@ function getFileNamePath(componentDirPath: string, componentName: string, ext: s
 }
 
 function gatherComponentsFiles(
-  components: Oas3Components,
+  components: Oas3ComponentsBase<Oas3Schema | Oas3_1Schema> | Oas3_1Components,
```

Comment:
Related to https://github.com/Redocly/redocly-cli/pull/1795/files?diff=split&w=1#r1898410733 :

```suggestion
  components: Oas3Components | Oas3_1Components,
```